REGRESSION:(294494@main) [ iOS ] 2x TestWebKitAPI.WritingTools.CompositionWith* (api-tests) are flaky failures

rr-codes opened the pull request:
#### 80ae0cca5a82ac7224175408e12e91758ffc2f7d
<pre>
REGRESSION:(294494@main) [ iOS ] 2x TestWebKitAPI.WritingTools.CompositionWith* (api-tests) are flaky failures
<a href="https://bugs.webkit.org/show_bug.cgi?id=298412">https://bugs.webkit.org/show_bug.cgi?id=298412</a>
<a href="https://rdar.apple.com/158343360">rdar://158343360</a>

Reviewed by Aditya Keerthi.

Add some more and slightly longer delays, consistent with the other test functions. Also remove an unnecessary `waitForValue`
lambda that was effectively just waiting 0.1s and then would fail via a JS exception in the failing case.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm:
(TEST(WritingTools, CompositionWithList)):
(TEST(WritingTools, CompositionWithTextAttachment)):

Canonical link: <a href="https://commits.webkit.org/299602@main">https://commits.webkit.org/299602@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/51b87619961c7347e334b57b719800c87aa9f7b9

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/119477 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/39169 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/29824 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/125748 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/71552 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/217a36cc-059c-4c2c-a5f9-95ac9e04fd9b) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/121354 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/39866 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/47749 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/90771 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/60079 "Passed tests") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/9d45e376-9c42-42f5-aba5-6283072a0c94) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/122429 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/31821 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/107127 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/71249 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/0889a774-0e02-43c4-aefb-a83215caf922) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/30859 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/25232 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/69395 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/101284 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/25424 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/128726 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/46400 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/35129 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/99357 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/46765 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/103321 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/99170 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25212 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/44611 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/22619 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/42964 "Built successfully") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/46261 "Built successfully") | | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/45726 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/49077 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/47413 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/914ce56d85f0c3c95786c4688a20a0474c8b406c. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50319

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/51b87619961c7347e334b57b719800c87aa9f7b9. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/50319

rr-codes committed:
REGRESSION:(294494@main) [ iOS ] 2x TestWebKitAPI.WritingTools.CompositionWith* (api-tests) are flaky failures
https://bugs.webkit.org/show_bug.cgi?id=298412
rdar://158343360

Reviewed by Aditya Keerthi.

Add some more and slightly longer delays, consistent with the other test functions. Also remove an unnecessary `waitForValue`
lambda that was effectively just waiting 0.1s and then would fail via a JS exception in the failing case.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/WritingTools.mm:
(TEST(WritingTools, CompositionWithList)):
(TEST(WritingTools, CompositionWithTextAttachment)):

Canonical link: https://commits.webkit.org/299602@main

webkit-commit-queue commented:
Committed 299602@main (80ae0cca5a82): <https://commits.webkit.org/299602@main>

Reviewed commits have been landed. Closing PR #50319 and removing active labels.
